docs/DeveloperGuide.adoc

```diff
@@ -334,26 +434,81 @@ Use case ends.
 +
 Use case ends.
 
-* 3a. The given index is invalid.
+* 3a. The one or more given parameters are invalid.
++
+[none]
+** 3a1. Application shows an error message.
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: Take notes for a patient
+
+*MSS*
+
+1.  User requests to open note GUI for a specific patient
+2.  User edit the patient’s notes
+3.  User request to exit note GUI
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1a. Invalid command
++
+[none]
+** 1a1. Application shows an error message.
++
+Use case ends.
+
+* 3a. Invalid command
 +
 [none]
-** 3a1. AddressBook shows an error message.
+** 3a1. Application shows an error message.
 +
 Use case resumes at step 2.
 
-_{More to be added}_
+[discrete]
+=== Use case: Adding an appointment
+
+*MSS*
+
+1.  User requests to list persons
+2.  Application shows a list of persons
+3.  User picks one patient from the list and add appointment date and time through command line
```

Comment:
The phrase "through command line" could be classified as internal details which could be unnecessary for use case.